docs/installation/mac.md

```diff
@@ -56,10 +57,37 @@ You can test if it's up and running by issuing the following command:
 redis-cli ping
 ```
 
+### Elasticsearch
+
+DEV requires Elasticsearch version 7 or higher.
+
+We recommend installing from archive, but if you prefer you can also use
+[Homebrew](https://brew.sh) to install Elasticsearch, directions at the bottom
+of this section. The following directions were
+[taken from the Elasticsearch docs themselves](https://www.elastic.co/guide/en/elasticsearch/reference/7.5/targz.html#install-macos),
+so check those out if you run into any issues or want further information.
+
+```shell
+wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.5.2-darwin-x86_64.tar.gz
+wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.5.2-darwin-x86_64.tar.gz
+shasum -a 512 -c elasticsearch-oss-7.5.2-darwin-x86_64.tar.gz.sha512
+tar -xzf elasticsearch-oss-7.5.2-darwin-x86_64.tar.gz
+cd elasticsearch-7.5.2/
+```
+
+To start elasticsearch:
+
+```shell
+./bin/elasticsearch
```

Comment:
maybe can mention that by using:

```
./bin/elasticsearch -d
```

the server can be "daemonized"

Reply:
good call